Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -38,4 +45,33 @@ public class ItemUseHandler : PacketHandler<GameSession> {
         
         session.Send(StoryBookPacket.Load(storyBookId));
     }
+
+    private static void HandleChatSticker(GameSession session, Item item)
+    {
+        string? htmlParameter = HttpUtility.HtmlDecode(item.Metadata.Function?.Parameters);
```

Comment:
We should write a util function for parsing these since I assume there are many cases that will need to do the same.

side note, why do we need to decode it as html, I thought it's just xml?